packages/react/package.json

```diff
@@ -0,0 +1,43 @@
+  {
+  "name": "@supabase/auth-helpers-react",
```

Comment:
Also need to change the neame in `packages/react/README.md`

Reply:
Yes for sure